src/deepforest/utilities.py

```diff
@@ -174,38 +174,64 @@ def convert_point_to_bbox(gdf: gpd.GeoDataFrame, buffer_size: float) -> gpd.GeoD
 
 def shapefile_to_annotations(
     shapefile: str | gpd.GeoDataFrame,
-    rgb: str | None = None,
     root_dir: str | None = None,
+    rgb: str | None = None,
     buffer_size: float | None = None,
     convert_point: bool = False,
+    label: str | None = None,
 ) -> gpd.GeoDataFrame:
-    """Convert shapefile annotations to DeepForest format.
+    raise DeprecationWarning(
+        "shapefile_to_annotations is deprecated, use read_file instead"
+    )
 
-    Args:
-        shapefile: Path to shapefile or GeoDataFrame
-        rgb: Path to RGB image
-        root_dir: Directory to prepend to image paths
-        buffer_size: Buffer size for point-to-polygon conversion
-        convert_point: Convert points to bounding boxes
+    if buffer_size is not None and convert_point:
+        raise DeprecationWarning(
+            "buffer_size argument is deprecated, use convert_point_to_bbox instead"
+        )
```

Comment:
Not sure if we need a deprecation warning for an argument inside a deprecated function